src/java/Light_vehicle.java

```diff
@@ -0,0 +1,36 @@
+package java;
+
+import java.util.Scanner;
+
+public class Light_vehicle extends Land_transport{
+    String TypeOfBody;
+    int NumberOfSeats;
+    private int travel;
+    private double a;
```

Comment:
@Hlieb что это за переменная  
`private double a;`? измени название, чтобы было понятно зачем она
```suggestion
    private double fuelConsumedAmount;
```